tests/test_wave_03.py

```diff
@@ -57,9 +57,12 @@ def test_friends_unique_movies_not_duplicated():
 
     # *************************************************************************************************
     # ****** Add assertions here to test that the correct movies are in friends_unique_movies **********
+
+    assert FANTASY_2 not in friends_unique_movies
```

Comment:
I would like to see more assertions here to truly test that `friend_unique_movies` do not have any duplicates, as it describes doing so on L47. It's customary for our test definition to describe in full detail what our test block intends to test. Currently, this assertion only says that `FANTASY_2` is not in friends_unique_movies, it doesn't actually test that friends_unique_movies has three distinct movies!